gno.land/pkg/gnoclient/integration_test.go

```diff
@@ -273,8 +287,15 @@ func TestSendMultiple_Integration(t *testing.T) {
 
 // Run tests
 func TestRunSingle_Integration(t *testing.T) {
+	// Setup packages
+	rootdir := gnoenv.RootDir()
+	config := integration.TestingMinimalNodeConfig(gnoenv.RootDir())
+	meta := loadpkgs(t, rootdir, "gno.land/p/demo/ufmt", "gno.land/r/demo/tests")
```

Comment:
Why do we load these specifically? 👀

Reply:
We need those packages for test (check bellow).